src/style.css

```diff
@@ -0,0 +1,88 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: bold;
+}
+
+.container input {
+  position: relative;
+  display: flex;
+
+  height: 70px;
+  margin-top: 20px;
+
+  font-family: Avenir, Arial, sans-serif;
+  font-weight: 300;
+
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+}
+
+.small-container input {
+  height: 42px;
+}
+
+.container input:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.container input:focus {
+  font-weight: bold;
+
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  box-shadow: none;
+}
+
+.input-field {
+  position: absolute;

```

Comment:
```suggestion
```
You don't need `position: absolute` if you're positioning this element using `padding-left` instead of `left`